cmd/openshift/operator/kodata/cabundles/config-service-cabundle.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
   name: config-service-cabundle
   namespace: openshift-pipelines
   labels:
-    operator.tekton.dev/release: v0.75.0
+    operator.tekton.dev/release: v0.75.1
```

Comment:
Any reason to update this version ?
Do we have any fix to have a patch release ?